### PR TITLE
Add -4, -6, --ipv4 CLI arguments, fixes #9

### LINKED ISSUE #9: -4 and -6 as CLI args?
I think this is pretty standard among CLI tools, and enables one to force v4. Thoughts?

### PR DESCRIPTION
**Pull request by silverwind**

### CODE REVIEW

**Comment by silverwind** [comment]
Hmm, might want to explicitely check for `-4`.

File: cli.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ var cli = meow({
 		'  $ public-ip',
 		'',
 		'Options',
-		'  --ipv6  Return the IPv6 address instead of IPv4',
+		'  -4, --ipv4  Return the IPv4 address',
```
Comment (sindresorhus) [inline cli.js]: Maybe add that it's the default?

```
'  -4, --ipv4  Return the IPv4 address (default)',
```

**Comment by silverwind** [comment]
updated

File: cli.js [hunk 2]
```diff
@@ -17,7 +18,14 @@ var cli = meow({
 	]
 });
```
Comment (sindresorhus) [inline cli.js]: You can actually pass `minimist` options as a third argument here.

```js
{
  alias: {
    4: 'ipv4',
    6: 'ipv6'
  }
}
```

File: cli.js [hunk 3]
```diff
@@ -17,7 +18,14 @@ var cli = meow({
...
 
-var fn = cli.flags.ipv6 ? 'v6' : 'v4';
+var fn;
+if (Object.keys(cli.flags).length === 0 || cli.flags['4'] || cli.flags.ipv4) {
```
Comment (sindresorhus) [inline cli.js]: `Object.keys(cli.flags).length === 0` is not a good check as it will break if we ever add more flags here.

Comment (silverwind) [inline cli.js]: Hmm, how would you suggest checking for no flags?

Comment (sindresorhus) [inline cli.js]: Since `ipv4` is default, you can just check the `ipv6` flag.

**Comment by silverwind** [comment]
okay, updated again. hopefully that nested ternary is readable :)

**Pull request closed without merging**